database/schema.sql

```diff
@@ -4,7 +4,7 @@ USE reviews;
 
 CREATE TABLE reviews(
   id int NOT NULL AUTO_INCREMENT,
-  listing_id int(4) NOT NULL,
+  listing_id varchar(2) NOT NULL,
```

Comment:
Is there a reason why `id` is an `int`, but `listing_id` is a `varchar(2)`? This seems inconsitent.